cmd/relayproxy/config/config.go

```diff
@@ -137,6 +138,9 @@ type Config struct {
 
 	// Version is the version of the relay-proxy
 	Version string
+
+	// APIKeys list of API keys that authorized to use endpoints
+	APIKeys map[string]bool `mapstructure:"apiKeys"`
```

Comment:
I am not sure to understand why you have a `map[string]bool`, is it to have a set here?
If so we may need to consider having a `map[string]interface{}` instead since it is a bit better in term of memory and size.

Reply:
thanks for the article, TIL
let me change it into slice since it's more easy to understand in config